diff --git a/node_modules/react-native-reanimated-skeleton/lib/ShiverBone.js b/node_modules/react-native-reanimated-skeleton/lib/ShiverBone.js
index 284be29..9a9ad97 100644
--- a/node_modules/react-native-reanimated-skeleton/lib/ShiverBone.js
+++ b/node_modules/react-native-reanimated-skeleton/lib/ShiverBone.js
@@ -1,6 +1,6 @@
 import React, { memo } from 'react';
 import { View, StyleSheet } from 'react-native';
-import LinearGradient from 'react-native-linear-gradient';
+import { LinearGradient } from "expo-linear-gradient";
 import Animated, { useAnimatedStyle, } from 'react-native-reanimated';
 import { useGetGradientTransform } from './worklets';
 import { useGetBoneStyles, useGetGradientSize, useGetGradientEndDirection, } from './hooks';
diff --git a/node_modules/react-native-reanimated-skeleton/src/ShiverBone.tsx b/node_modules/react-native-reanimated-skeleton/src/ShiverBone.tsx
index 557fcee..fd9e91a 100644
--- a/node_modules/react-native-reanimated-skeleton/src/ShiverBone.tsx
+++ b/node_modules/react-native-reanimated-skeleton/src/ShiverBone.tsx
@@ -1,6 +1,6 @@
 import React, { memo } from 'react';
 import { View, StyleSheet } from 'react-native';
-import LinearGradient from 'react-native-linear-gradient';
+import { LinearGradient } from 'expo-linear-gradient';
 import Animated, {
   SharedValue,
   useAnimatedStyle,
